# Patch to make grass-5.7.0.tar.gz compile on Fedora2.
# Necessary until RT bug #2526 is solved.
--- configure.orig	2004-06-17 15:26:22.000000000 +0200
+++ configure	2004-07-01 17:02:03.119838488 +0200
@@ -8085,7 +8085,7 @@
 ac_lib_var=`echo readline'_'readline | sed 'y%./+-%__p_%'`
 
 ac_save_LIBS="$LIBS"
-LIBS="-lreadline  $LIBS"
+LIBS="-lreadline -lncurses  $LIBS"
 cat > conftest.$ac_ext <<EOF
 #line 8091 "configure"
 #include "confdefs.h"
@@ -8112,7 +8112,7 @@
 
 if eval "test \"`echo '$ac_cv_lib_'$ac_lib_var`\" = yes"; then
   echo "$ac_t""yes" 1>&6
-  READLINELIB="$READLINELIB -lreadline "
+  READLINELIB="$READLINELIB -lreadline -lncurses "
 else
   echo "$ac_t""no" 1>&6
 
